manpages/lumina-mediaplayer.8

```diff
@@ -1,43 +1,60 @@
-.Dd October 27, 2017
+.Dd October 31, 2017
 .Dt LUMINA-MEDIAPLAYER 8
 .Os 1.3.3
 
 .Sh NAME
 .Nm lumina-mediaplayer
-.Nd is a graphical utility to play media files and stream online radio services.
+.Nd a graphical utility to play media files and stream online radio
+services.
 
 .Sh SYNOPSIS
 .Nm
-.Op <FileName1 FileName2  ... >
-.Nm
-.Op <argument>
+.Op Fl -pandora
+.Op Ar FILES
+
 
 .Sh DESCRIPTION
 .Nm
-opens up the graphical interface for the media player. The user can sync a Pandora account
-and stream music. The user can also create and manage a playlist of media files from their system.
+opens up the graphical interface for the media player.
+The user can sync a Pandora account and stream music.
+The user can also create and manage a playlist of media files from their
+system.
 .Pp
-The command can take multiple file names at a time and add them into a playlist. It can
-open .mp3, .mp4, .flac, .acc, .mpeg2, and mpeg4 files.
-Other than using the gui, the user is able to add a song by specifying a file name
-or multiple file names.
+The command can take multiple file names at a time and add them into a
+playlist.
+It can open .mp3, .mp4, .flac, .acc, .mpeg2, and mpeg4 files.
+Other than using the gui, the user is able to add a song by specifying
+a file name or multiple file names.
 .Pp
 .Nm
-has one optional argument, and it simply opens up the interface on the Pandora account page.
+has one optional argument:
 .Bl -tag -width indent
 .It Ic --pandora
 Opens the interface on the Pandora account page.
+.El
 
 .Sh EXAMPLES
+.Pp
+.Dl % lumina-mediaplayer Downloads/Testing.mp3
+Opens the interface and adds Testing.mp3 into the playlist.
+.Dl % lumina-mediaplayer Downloads/Testing.mp3 Testing2.mp4
+Opens the interface and adds Testing.mp3 and Testing2.mp4 into the
+playlist.
 
+.Sh FILES
 .Bl -tag -width indent
-.It lumina-mediaplayer Downloads/Testing.mp3
-Opens the interface and adds Testing.mp3 into the playlist.
-.It lumina-mediaplayer Downloads/Testing.mp3 Testing2.mp4 Testing 3.mp3
-Opens the interface and adds Testing.mp3, Testing2.mp4, and
-Testing 3.mp3 into the playlist.
-.It lumina-mediaplayer --pandora
-Opens the interface on the Pandora account page.
+.It Pa /usr/local/bin/lumina-mediaplayer
+.El
+
+.Sh OPTIONAL RUNTIME DEPENDENCIES
+If other lumina utilities are installed more options are available.
+The following are optional for
```

Comment:
pianobar isn't a Lumina specific utility, so we should rework this sentence. -> Installing other utilities can make more options available. These extra utilities are supported: <begin list>